lib/stripe_mock/request_handlers/customers.rb

```diff
@@ -61,7 +61,7 @@ def new_customer(route, method_url, params, headers)
       def update_customer(route, method_url, params, headers)
         stripe_account = headers && headers[:stripe_account] || Stripe.api_key
         route =~ method_url
-        cus = assert_existence :customer, $1, customers[stripe_account][$1]
```

Comment:
This failed `undefined method `[]' for nil:NilClass` instead of `Stripe::InvalidRequestError` when looking for a customer in a stripe account